src/main/kotlin/com/wgslfuzz/core/ShaderJob.kt

```diff
@@ -88,6 +88,8 @@ class ShaderJob(
         value: Expression,
         offset: Int,
     ): List<Int> {
+        // The resolved type may be a reference. We only care about the store type of the reference, so remove the outer reference.
```

Comment:
Similar to the comment below, we shouldn't find a reference here (I think!), so removing this and relying on the `else` branch to tell us if I'm wrong is the way to go.